xmppserver/src/main/java/org/jivesoftware/openfire/XMPPServer.java

```diff
@@ -769,6 +770,7 @@ private void loadModules() {
         loadModule(OfflineMessageStrategy.class.getName());
         loadModule(OfflineMessageStore.class.getName());
         loadModule(VCardManager.class.getName());
+        loadModule(SoftwareVersionManager.class.getName());
```

Comment:
This new module is not a 'core' module. Please move it to the bottom of the list of the 'standard' modules.